specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/networkProfile.json

```diff
@@ -0,0 +1,486 @@
+        "description": "Gets all the network profiles in a subscription.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of NetworkProfile resources.",
+            "schema": {
+              "$ref": "#/definitions/NetworkProfileListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List all network profilees": {
+            "$ref": "./examples/NetworkProfileListAll.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/networkProfiles": {
+      "get": {
+        "tags": [
+          "NetworkProfiles"
+        ],
+        "operationId": "NetworkProfiles_List",
+        "description": "Gets all network profiles in a resource group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Request successful. The operation returns a list of NetworkProfile resources.",
+            "schema": {
+              "$ref": "#/definitions/NetworkProfileListResult"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List resource group network profilees": {
+            "$ref": "./examples/NetworkProfileList.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NetworkProfilePropertiesFormat": {
+      "properties": {
+        "containerNetworkInterfaces": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ContainerNetworkInterface"
+          },
+          "description": "List of child container network interfaces."
+        },
+        "containerNetworkInterfaceConfigurations": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ContainerNetworkInterfaceConfiguration"
+          },
+          "description": "List of chid container network interface configurations."
+        }
+      },
+      "description": "Network profile properties."
+    },
+    "NetworkProfile": {
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/NetworkProfilePropertiesFormat",
```

Comment:
You probably want to use x-ms-client-flatten to make better SDK.
Example:
https://github.com/Azure/azure-rest-api-specs/blob/32b0d873aa851d456dfde7d6ba1d89ff33f897f0/specification/network/resource-manager/Microsoft.Network/stable/2016-12-01/applicationGateway.json#L511